lib/livebook/config.ex

```diff
@@ -493,4 +513,11 @@ defmodule Livebook.Config do
     IO.puts("\nERROR!!! [Livebook] " <> message)
     System.halt(1)
   end
+
+  @doc """
+  Parses zero trust identity provider from env.
+  """
+  def identity_provider!(env) do
+    System.get_env(env)
```

Comment:
Let's move the parsing here and store the tuple in the configuration.

Reply:
Agreed!